content/news/release-20161019.md

```diff
@@ -0,0 +1,65 @@
+CRITICALLY WEAKENED.  REPORTS INDICATE THE PRESENCE OF A MULTI-PLAYER
+DESYNC CRASH IN THE RED ALERT BATTLEGROUND FOLLOWING ENEMY DEPLOYMENT
+OF GPS SATELLITES.  SCRAMBLED LOBBY GAME SPEED COMMANDS ARE CONNECTED
+TO THE REDUCED FIELD MOBILITY OF SOME COMMANDERS.
+
+PROCEEED TO <a href="/download/">www.openra.net/download/</a> TO OBTAIN THE FORTIFIED RELEASE.
+RENDEZVOUS WITH OTHER COMMANDERS IN THE MULTIPLAYER LOBBY AND DESTROY
+ALL ENEMY OPPOSITION.
+
+TRANSMISSION ENDS.
+</pre>
+</div>
+
+A release-20161019 hotfix is now available to fix two critical multi-player bugs that were missed during the playtesting phase. The original release-20161015 release announcement is included below:
+
+<hr />
+Today we are happy to announce the latest stable release of OpenRA!
+
+While this cycle has seen relatively little changes to the core game code with just about 28,000 changed lines, the new features and fixes it yields make up for it by being very visible.
+
+The first place you will notice this is the much improved game content installer. It is now able to copy and install game assets from many more installation sources, such as the original release CDs, the First Decade DVDs, installed copies of the Dune 2000 Gruntsmod edition and even the C&C Ultimate Edition from Origin. Additionally, for the first time ever, the content installer will now also automatically install the campaign briefing videos, which means you no longer have to do that manually.
+
+<div style="text-align:center" markdown="1">
+![Game content installer](/images/news/20161015-ra-contentinstaller.png)
+</div>
+
+This features comes with one minor down-side: since the new assets are installed into a different place than in previous versions of OpenRA, every player is required to install the game assets again when launching this new version for the first time.
+
+As usual, our quest to complete porting the original campaign missions to OpenRA is ongoing. This time around, there are seven new missions for Tiberian Dawn, and one each for Red Alert and Dune 2000. For Red Alert, there are also three new missions which can be played in both single- and cooperative multiplayer mode and which long-time players may find to be vaguely familiar.
+
+Other new features and fixes include the following:
+
+* Fixes to a lot of crashes.
+* Fixes to a number of bugs that impacted game servers, such as the infamous "grey ping bug".
+* New configuration options for customizing mouse scrolling behaviour.
+* New hotkeys for navigating to map edges, and bookmarking positions on the map.
+* New hotkeys for controlling the music player, as well as producing five units at once.
+* Improved aircraft reloading behaviour.
+
+There are many more changes than we are able to mention here, including the usual round of balance changes. Head on over to the [Changelog page on our wiki](https://github.com/OpenRA/OpenRA/wiki/Changelog/561383b9ba791a557cde37616f42a8df7d8a3182) to see the complete list.
```

Comment:
Oh, you might want to link to the updated changelog as well: https://github.com/OpenRA/OpenRA/wiki/Changelog/0ae64ef1b6dded0a55900ab67561c2d56885c461